enas/cifar10/main.py

```diff
@@ -3,14 +3,16 @@
 from __future__ import print_function
 
 import os
-import cPickle as pickle
+import _pickle as pickle
 import shutil
 import sys
 import time
 
 import numpy as np
 import tensorflow as tf
 
+sys.path.append('./')
```

Comment:
One way to not require this is to create a [setup.py](http://the-hitchhikers-guide-to-packaging.readthedocs.io/en/latest/creation.html#setup-py-description), then install with the `-e` option.

```
pip install -e . --user --upgrade
```

```
pip3 install -e . --user --upgrade
```

the keras code's setup.py is a pretty simple example